kivymd/uix/progressbar.py

```diff
@@ -288,7 +288,9 @@ def _create_determinate_animations(self):
         )
         self.running_anim.bind(on_complete=self.catching_up)
         self.catching_anim = Animation(
-            opacity=0, t=self.catching_transition, d=self.catching_duration,
+            opacity=0,
+            t=self.catching_transition,
+            d=self.catching_duration,
         )
         self.catching_anim.bind(on_complete=self.running_away)
 
```

Comment:
Updated by pre-commit of github actions. 8994a39cd2b1050aa6b8342b533d148cbe2e065a